app/views/clearance_mailer/change_password.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= t(".opening") %>
+<p><%= t(".opening") %></p>
 
-<%= edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %>
+<p><%= link_to 'Change my password', edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %></p>
```

Comment:
Can we use a locale for this text? (the rest of the email text is from I18n so that seems like it would be good to stay consistent)